honestcash-v2/src/app/auth/services/auth.service.ts

```diff
@@ -59,90 +54,36 @@ export class AuthService {
     @Inject(LocalStorageToken) private localStorage: Storage,
     private store: Store<AppStates>,
     private http: HttpService,
+    private walletService: WalletService,
   ) {
     this.isPlatformBrowser = isPlatformBrowser(this.platformId);
   }
 
-  public getToken(): string {
```

Comment:
these functions are abstracted into their own service such that:
- token, userId relating stuff were something relating to user so it was abstracted into userService,
- wallet related stuff were abstracted into walletService
so Auth module isAuthenticated is only a shortcut so that we don't check for existence of both (or that we forget) to know whether token and mnemonic exists. at UserSetup, WalletSetup, UserCleanup, WalletCleanup, AuthService.unauthenticate and .authenticate is used to toggle between states